src/js/dataMapping/recipients/topCategories.js

```diff
@@ -18,7 +18,7 @@ export const recipientCategoryTitles = {
     awarding_agency: 'Awarding Agencies',
     awarding_subagency: 'Awarding Sub-Agencies',
     federal_account: 'Federal Accounts',
-    cfda: 'CFDA Programs',
+    assistance_listings: 'Assistance Listings (CFDA Programs)',
```

Comment:
I would try changing this key back to cfda and see if that fixes the issue with the data loading in the table

Reply:
fixed